pycytominer/cyto_utils/load.py

```diff
@@ -1,8 +1,13 @@
+"""
+Module for loading profiles from files or dataframes.
+"""
+
 import csv
 import gzip
 import pathlib
-from typing import Union
+from typing import Any, Optional, Union
 
+import anndata as ad
```

Comment:
It is a big deal for pycytominer to now require `anndata` as a dependency. I think this is fine to do, but I want to confirm that you've considered the consequences (both good and bad).

A good is supporting interoperability, while a bad is the extra maintenance it may require (I do not the extent of additional maintenance, and it could be minimal). Another potential, maybe "neutral" consequence, is that an astute observer may see `anndata` as a dependency and then assume it is Pycytominer's intended default. It very may well become the default (only benchmarking and time will tell), but I don't think it is currently.

Reply:
Thinking about this more I wondered if and how much this might become a stronger constraint to installs where folks might not be using anndata in their work (we haven't required it upfront before this point so it could take some time for this to build within the community as a pattern). I opted to create an `anndata` extra for installation following existing conventions within the project and using a TypeVar to help simulate the type where needed for `load`.